src/main/java/org/springframework/data/repository/query/ExtensionAwareEvaluationContextProvider.java

```diff
@@ -424,25 +424,25 @@ public EvaluationContextExtensionAdapter(EvaluationContextExtension extension,
 		/**
 		 * Returns the extension identifier.
 		 * 
-		 * @return
+		 * @return the id of the extension
 		 */
-		public String getExtensionId() {
+		String getExtensionId() {
 			return extension.getExtensionId();
 		}
 
 		/**
 		 * Returns all functions exposed.
 		 * 
-		 * @return
+		 * @return all exposed functions.
 		 */
-		public Map<String, Function> getFunctions() {
+		FunctionsMap getFunctions() {
```

Comment:
This is just in a private class anyway. A separate interface sounds like overkill to me in this case.